wrangling.Rmd

```diff
@@ -1343,7 +1315,7 @@ region_lang |>
 > the use of the `summarize` + `across` functions paired 
```

Comment:
I think above this line it says "mix" instead of min:
"Similarly to when we use base R statistical summary functions (e.g., max, **mix**, mean, sum, etc)"

Reply:
fixed all mix vs min